src/Security.tsx

```diff
@@ -51,7 +56,11 @@ const Security: React.FC<{
     };
 
     // Add okta-react userAgent
-    oktaAuth.userAgent = `${process.env.PACKAGE_NAME}/${process.env.PACKAGE_VERSION} ${oktaAuth.userAgent}`;
+    if (oktaAuth._oktaUserAgent) {
+      oktaAuth._oktaUserAgent.addEnvironment(`${process.env.PACKAGE_NAME}/${process.env.PACKAGE_VERSION}`);
+    } else {
+      console.warn('_oktaUserAgent is not available on auth SDK instance. Please use okta-auth-js@^5.3.1 .');
+    }
```

Comment:
@oleksandrpravosudko-okta 
This change has introduced a pretty annoying bug that manifests when the app is reloaded via HMR. not sure why `oktaAuth` needs to be mutated to add the okta-react as userAgent but there will now be two agents when this [runs](https://github.com/okta/okta-react/pull/159/files#diff-d1d964ff54a1f94b6ca7e705618235cfa17270216413eb94a12c757ea8f9e49fR60):
```
oktaAuth.userAgent
"@okta/okta-react/6.1.0 okta-auth-js/5.3.1"
```
This will then break the check for `oktaAuthMajorVersion` on [line 94](https://github.com/okta/okta-react/pull/159/files#diff-d1d964ff54a1f94b6ca7e705618235cfa17270216413eb94a12c757ea8f9e49fR94) previously added by @shuowu because `oktaAuthMajorVersion` will now evaluate to `"okta-react"` whereas `process.env.AUTH_JS_MAJOR_VERSION ` evaluates to `5`... `"okta-react" === 5 -> false`